__tests__/util.test.ts

```diff
@@ -207,6 +207,19 @@ describe('isValidURL', () => {
   });
 });
 
+describe('isNotValidURL', () => {
+  test.each([
+    ['https://github.com/docker/buildx.git', false],
+    ['https://github.com/docker/buildx.git#refs/pull/648/head', false],
+    ['git@github.com:moby/buildkit.git', true],
+    ['git://github.com/user/repo.git', true],
+    ['github.com/moby/buildkit.git#main', true],
+    ['v0.4.1', true]
+  ])('given %p', async (url, expected) => {
+    expect(Util.isValidURL(url)).toEqual(expected);
+  });
+});
```

Comment:
This is already covered by `isValidURL`

Reply:
hey max i am new to open sourcing went ahead just open the repo and tried to contribute something to the community 
i know my code won't any new ground breaking chages or bring anything i did cause it it wont change anything.
If possible merge it or ill try to contribute for some other thing

Reply:
> hey max i am new to open sourcing went ahead just open the repo and tried to contribute something to the community

That's perfectly fine and thanks for taking a step in the open source community and contributing to this project!

> i did cause it it wont change anything.
> If possible merge it or ill try to contribute for some other thing

Well that's actually an issue. If your change does not have any purpose but just for the sake of adding more code, it won't be approved. It needs to have a relevant purpose (bug fix, enhancement, docs, ...).

Reply:
I actually went there and reached out to bunch of people every one is busy with their own work and some are charging way to high as your also an developer in the space i thought u could help that it Max nothing else